### PR TITLE
Adding the heat map visualization. Also added a dropdrown list

### PR DESCRIPTION
**Pull request by sayaji15**
to select destination gw.

### CODE REVIEW

**Comment by sayaji15** [comment]
@nxanil Created PR off the master branch

**Comment by nxanil** [comment]
@sayaji15 - Please explain the task related to Dropdown list, as I am not able to see it further..

File: public/configurations/visualizations/aar-flow-sla-heatmap.json [hunk 1]
```diff
@@ -24,5 +24,21 @@
             { "column": "slastatus", "label": "Status" }
         ]
     },
+    "filterOptions": {
```
Comment (ronakmshah) [inline public/configurations/visualizations/aar-flow-sla-heatmap.json]: @nxanil this snippet needs to change to something like:

"filterOptions": {{ nsgListFilterOptions }}

Where nsgListFilterOptions is a variable which is passed around in context.

Psudocode could be like:
'''
nsgListFilterOptions = {"Destination GW": { "parameter": "dnsg"}}
for nsg in nsgList: {
    nsgListFilterOptions['Destination GW']['options'].append({ 'label':  nsg, 'value': nsg})
}
nsgListFilterOptions['Destination GW']['default'] = nsg
'''

Thus you create nsgListFilterOptions dynamically. @sayaji15  has mentioned where we populate NSG list in our framework. so you can create this variable there. Also, it is imp to make sure that it changes whenever that list is modified. 

Hope that clears.

**Comment by ronakmshah** [comment]
@nxanil please see my comment above.

**Comment by nxanil** [comment]
@ronakmshah @sayaji15 - Any particalar python script that I can use to load data for aarEnterprise, I can find any relevant one?

Thanks!

**Comment by nxanil** [comment]
@sayaji15 @ronakmshah 
I have commited the few changes please verify and do let me know if you have any issues.

**Pull request merged**